### DISCUSSION TITLE
Cannot specify style setter with multibinding

### DISCUSSION THREAD

**Discussion by wgraaf** [AvaloniaUI/Avalonia#16911] [Q&A] [answered]
I have  a menu item of which the sub-items are populated from an ObservableCollecttion<string> ColumnVisibilityDescriptors:

```
      <MenuItem Name="ColumnsMenuItem" Header="Columns" ItemsSource="{Binding ColumnVisibilityDescriptors}"}>
        <MenuItem.Styles>
          <Style Selector="MenuItem">
            <Setter Property="Header" Value="{Binding, Converter={x:Static vm:ColumnVisibilityConverter.MenuHeader}}"/>
          </Style>
        </MenuItem.Styles>
      </MenuItem>

```

So far, so good; the MenuHeader converter renders the sub-headers as expected from the binding strings.
I want to enable the sub-menu items  based on two parameters, using  a MultiValueConverter.
I cannot seem to specify this in the axaml.
I tried:

```
      <MenuItem Name="ColumnsMenuItem" Header="Columns" ItemsSource="{Binding ColumnVisibilityDescriptors}"}>
        <MenuItem.Styles>
          <Style Selector="MenuItem">
            <Setter Property="Header" Value="{Binding, Converter={x:Static vm:ColumnVisibilityConverter.MenuHeader}}"/>
            <Setter Property="IsEnabled">
              <Setter.Value>
                <MultiBinding Converter="{x:Static vm:ColumnVisibilityConverter.IsMenuEnabled}">
                  <Binding Path="{Binding}"/>
                  <Binding Path="ViewType"/>
                </MultiBinding>
              </Setter.Value>
            </Setter>
          </Style>
        </MenuItem.Styles>
      </MenuItem>

```

which gives me compile error AVLN3000:
`Unable to find suitable setter or adder for property Path of type Avalonia.Markup.Xaml:Avalonia.Markup.Xaml.MarkupExtensions.ReflectionBindingExtension for argument Avalonia.Markup:Avalonia.Data.Binding, available setter parameter lists are:`

I get the same result using a style class instead of an inline style.
Is there a way to achieve this?

**Comment by workgroupengineering** [comment, marked as answer]
First error is in this line

```xaml
<Binding Path="{Binding}"/>
```

To binding current item should be:

```xaml
<Binding Path="."/>
```
What is ViewType? Is it a constant value or a field?

**Reply by wgraaf** [reply]
ViewType is a (string) field
